lesson29/src/js/utility/checktoken.js

```diff
@@ -2,7 +2,11 @@ const urlParameter = Object.fromEntries(new URLSearchParams(window.location.sear
 const currentPageToken = urlParameter.token;
 const registeredToken = localStorage.getItem("token");
 
-if(currentPageToken !== registeredToken) {
+if (!registeredToken || !currentPageToken){
+    window.location.href = "./../notautherize.html";
+}
+
+if (registeredToken && currentPageToken !== registeredToken) {
```

Comment:
Q. 
`registeredToken`は上部の条件式でtrueであることをがわかってるので不要だと思いましたが
必要ですか？

```suggestion
if (currentPageToken !== registeredToken) {
```

Reply:
いつもレビューありがとうございます！
その通りでした！
不要なので削除しました。
[7d08802](https://github.com/haru-programming/JavaScript-lessons/pull/40/commits/7d08802404d321716071cba3024471a28a025c13)